CHANGELOG.md

```diff
@@ -201,6 +221,8 @@ The following are deprecated and will be removed in a future release. For anythi
 - InputFieldGroup.visible deprecated since 4.27. Use GroupInput.visible instead.
 - InputFieldGroup deprecated since 4.27. Use GroupInput instead.
 - LayerList.iconClass deprecated since 4.27. Use icon instead.
+- LayerList.multipleSelectionEnabled depricated since 4.29. Use selectionMode instead.
+- LayerList.selectionEnabled depricated since 4.29. Use dragEnabled and selectionMode instead.
```

Comment:
```suggestion
- LayerList.multipleSelectionEnabled deprecated since 4.29. Use selectionMode instead.
- LayerList.selectionEnabled deprecated since 4.29. Use dragEnabled and selectionMode instead.
```